### DISCUSSION TITLE
Reset usage

### DISCUSSION THREAD

**Discussion by konstTSK** [mobinjavari/v2ray-api-php#34] [Q&A] [answered]
![reset1](https://github.com/mobinjavari/v2ray-api-php/assets/33707789/c2349ffa-7671-4ff8-b3c9-9a124e12aca7)

print_r($xui->update($update, $where));

not working resert traffic

**Comment by mobinjavari** [comment]
Hello, what panel are you using?

**Reply by konstTSK** [reply]
i use  3X-UI [v2.1.3]  Xray v1.8.6 

![изображение](https://github.com/mobinjavari/v2ray-api-php/assets/33707789/cb24c50d-4f51-471d-a144-d80534edfaa0)
![изображение](https://github.com/mobinjavari/v2ray-api-php/assets/33707789/cb167a59-9a69-4a16-bed3-36682fb2cdd7)

**Reply by konstTSK** [reply]
nothing happens, the traffic should be reset to zero

**Comment by mobinjavari** [comment, marked as answer]
Yes, nothing is considered to reset the traffic for the Senai panel in the eternity function

This problem is solved 🤍

**Reply by konstTSK** [reply]
Thank you for fixing.
